### PR TITLE
fix: remove check inside setUid to check if uid exists - RKT-8671

### PR DESCRIPTION
**Pull request by timcubb**
RKT-8671

### CODE REVIEW

**Comment by timcubb** [comment]
hotfix-v1.7-v1.8

File: src/webloader.js [hunk 1]
```diff
@@ -6,9 +6,13 @@ import { modes } from './modes/index.js';
 import { setCookie, getCookie } from "./cookies.js";
 
 
+function getId(evolv, key) {
```
Comment (robertsevernsentient) [inline src/webloader.js:9]: This should return the id -- right now it returns the context

**Review by robertsevernsentient — CHANGES_REQUESTED** [review]

File: src/webloader.js (hunk 1 above)
Comment (timcubb) [reply src/webloader.js:9]: Fixed - also found a bug where we are checking if the `key` exists in `evolv.context` and then using `evolv[key]` to get the `id` - should be `evolv.context[key]`

**Comment by timcubb** [comment]
@robertsevernsentient - PR reverted and now removing check in setUid()

**Review by robertsevernsentient — APPROVED** [review]

**Pull request merged**